compiler/vm/vmgen.nim

```diff
@@ -127,15 +127,25 @@ type
 
   BlockKind = enum
     bkBlock   ## labeled block
-    bkTry     ## the ``try`` and ``except`` clause of a ``finally``-having
-              ## try statement
     bkExcept  ## ``except`` clause
     bkFinally ## ``finally`` clause
 
+  BlockInfo = object
+    label: BlockId
+    oldRegisterCount: int
+      ## upper bound of allocated registers at the beginning of the block
```

Comment:
```suggestion
    oldRegisterCount: int
      ## upper bound of allocated registers at the beginning of the block
    label: BlockId
```
Really inconsequential, but due to alignment one could save 8 bytes (on 64bit archs) by swapping these two fields.

Reply:
It doesn't hurt to swap them. But yeah, one would need structured block-like nesting in the millions for this to be noticeable :D